core/frontend/src/tile/map/MapCartoRectangle.ts

```diff
@@ -15,7 +15,7 @@ const scratchMercatorFractionRange = Range2d.createNull();
 const scratchPoint2d = Point2d.createZero();
 
 /** A specialization of Range2d that represents a cartographic range used by map tiles.
- * @internal
+ * @public
  */
 export class MapCartoRectangle extends Range2d {
```

Comment:
Suggestion: There is a class named CartographicRange. Maybe that is what we should use here instead of creating a new class for representing a range.

Reply:
@MarcBedard8 I will keep getMapLayersRange and MapCartoRectagle private for now, lets see how we can merge both types later.

Reply:
ok